SWLOR.Game.Server/Feature/GuiDefinition/AreaManager.cs

```diff
@@ -0,0 +1,361 @@
+﻿using SWLOR.Game.Server.Core.Beamdog;
+using SWLOR.Game.Server.Feature.GuiDefinition.ViewModel;
+using SWLOR.Game.Server.Service.GuiService;
+
+namespace SWLOR.Game.Server.Feature.GuiDefinition
+{
+    public class AreaManagerDefinition : IGuiWindowDefinition
+    {
+        private readonly GuiWindowBuilder<AreaManagerViewModel> _builder = new();
+
+        public GuiConstructedWindow BuildWindow()
+        {
+            _builder.CreateWindow(GuiWindowType.AreaManager)
+                .SetIsResizable(true)
+                .SetIsCollapsible(true)
+                .SetInitialGeometry(0, 0, 975f, 650f)
```

Comment:
May need to reduce the default size of this window. On smaller screens it'll be difficult/impossible to select the resize section of the window. Bit of a quirk with NUI. It's safer to pick a smaller default and let the player do the resizing, especially when their UI is scaled.